lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -35,6 +37,25 @@ def ask_questions
       confirm
     end
 
+    def choose_disk
+      @disk = ask_for_disk("database disk")
+    end
+
+    def initialize_postgresql_disk
```

Comment:
Ah yes. Well then, since refactoring `LogicalVolumeManagement` seems to be beyond the scope of this PR, perhaps leaving `initialize_postgresql_disk` here but making it private?